profiles/hemantwasthere.json

```
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/51391473/191279633-3cb321f7-af64-4743-bc6a-a590a5081afa.png",
+  "issueId": 1385,
+  "name": "Hemant",
+  "username": "hemantwasthere"
+}
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```